tools/image_processing/imagej2/imagej2_macros.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="3.10.2">atlas</requirement>
+            <requirement type="package" version="20141125">fiji</requirement>
+            <requirement type="package" version="1.0.11">javabridge</requirement>
+            <requirement type="package" version="1.9">numpy</requirement>
+            <requirement type="package" version="1.0.4">python_bioformats</requirement>
+        </requirements>
+    </xml>
+    <xml name="stdio">
+        <stdio>
+            <exit_code range="1:"/>
+            <exit_code range=":-1"/>
+            <regex match="Error:"/>
+            <regex match="Exception:"/>
+        </stdio>
+    </xml>
+  	<xml name="max_heap_size_type_conditional">
+        <conditional name="set_max_heap_size">
+            <param name="max_heap_size_type" type="select" label="Maximum size of the memory allocation pool used by the JVM" help="This value must be a multiple of 256k or it will be ignored and the system default will be used.">
+                <option value="default" selected="True">Do not set</option>
+                <option value="kilobytes">Set in kilobytes</option>
+                <option value="megabytes">Set in megabytes</option>
+            </param>
+            <when value="default">
+                <param name="max_heap_size" type="integer" value="0" label="Do not set" help="Use system default"/>
+            </when>
+            <when value="kilobytes">
+                <param name="max_heap_size" type="integer" value="512" min="256" label="Maximum size, in kilobytes, of the memory allocation pool" help="Examples: 256, 512, etc."/>
+            </when>
+            <when value="megabytes">
+                <param name="max_heap_size" type="integer" value="512" min="256" label="Maximum size, in megabytes, of the memory allocation pool" help="Examples: 256, 512, etc."/>
+            </when>
+        </conditional>
+    </xml>
+	<token name="@HELP_CITATIONS@">
+
+------
+
+**Citation**
+
+If you use this tool in Galaxy, please cite Blankenberg D, et al. *In preparation.*
```

Comment:
samtools_sort.xml is the only one in galaxytools, but it is in gatk2_macros.xml in tools-iuc and in many tools in tools-devteam.